src/routers/api/v1/posts.ts

```diff
@@ -50,6 +60,60 @@ router.post("/", koaBody(), async ctx => {
         } else {
             post.files = []
         }
+        const replies = new Set(post.text.match(repliesRegex))
+        for (const reply of replies) {
+            const target = await transactionalEntityManager.findOne(User, {
+                screenName: reply.replace("@", ""),
+            })
+            if (target != null) {
+                const subscriptions = await transactionalEntityManager.find(Subscription, {
+                    user: target,
+                    revokedAt: null,
+                })
+                for (const subscription of subscriptions) {
+                    const subscriptionOptions = {
+                        endpoint: subscription.endpoint,
+                        keys: {
+                            p256dh: subscription.publicKey,
+                            auth: subscription.authenticationSecret,
+                        },
+                    }
+                    const icon = post.user.avatarFile
+                        ? post.user.avatarFile.variants
+                              .filter(variant => variant.type == "thumbnail")
+                              .sort(variant => variant.score)[0]
+                        : null
+                    const payload = {
+                        title: `${post.user.name} (@${post.user.screenName})`,
+                        body: post.text,
+                        icon: icon,
```

Comment:
type: mention みたいなのがあるとよさそう (今後的に)
```suggestion
                        icon: icon,
                        type: "mention",
```

Reply:
追加した